src/groupByKey.ts

```diff
@@ -2,6 +2,24 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+type ObjectForSorting = {
```

Comment:
```suggestion
type ObjectForSorting = Record<string, string | number>
```
There are records in the typescript for such purpose